### PR TITLE
Remove the second time break on parent function.

### LINKED ISSUE #9: Remove the second time stop on the parent function.
Consider the case, we set two breakpoints as following:

```javascript
function HelloDebugger() {
  var o = {     // <--- set breakpoint here
    a : 123,
    b : 666
  };
}

function AAA() { return HelloDebugger(); }

function BBB() {
  return AAA(); // <--- set breakpoint here
}

BBB();
```

Currently, we will break on `return AAA();` for second time after we break inside `HelloDebugger()`.

This seems OK if we run the in the plain code.

Consider the `for`-loop, this behavior will confuse users.

@mostafa

### PR DESCRIPTION
**Pull request by panzhongxian**
Fix #9

### CODE REVIEW

**Comment by mostafa** [comment]
Can you add it as a test case to https://github.com/mostafa/goja/blob/debugger/debugger_test.go?

**Comment by panzhongxian** [comment]
@mostafa Test case has been added.

File: debugger_test.go [hunk 1]
```diff
@@ -18,21 +18,16 @@ func TestDebuggerBreakpoint(t *testing.T) {
...
+	setBreakpointAndLog := func(line int) {
+		if err := debugger.SetBreakpoint("test.js", line); err != nil {
+			t.Fatal(err)
+		} else {
+			t.Logf("Set breakpoint on line %d", line)
+		}
 	}
+	setBreakpointAndLog(3)
+	setBreakpointAndLog(4)
+	setBreakpointAndLog(5)
```
Comment (mostafa) [inline debugger_test.go:30]: Nice! 👏

File: debugger_test.go [hunk 2]
```diff
@@ -184,6 +177,58 @@ func TestDebuggerContinue(t *testing.T) {
...
+				t.Fatalf("expect line: %d, wrong line: %d", line, debugger.Line())
+			} else {
+				t.Logf("hit breakpoint on line %d", debugger.Line())
+			}
+		}
+		/*
+			reason := debugger.Continue()
+			// unreachable?
+			t.Fatalf("wrong activation %s", reason)
+		*/
```
Comment (mostafa) [inline debugger_test.go:226]: If this is not needed, you might want to remove it.

**Review by mostafa — APPROVED** [review]
I think rebasing from latest master would cause the extra commits to go away. Otherwise LGTM.

**Comment by panzhongxian** [comment]
> I think rebasing from latest master would cause the extra commits to go away. Otherwise LGTM.

@mostafa I have rebased it and removed the useless comment.

**Pull request merged**